README.md

```diff
@@ -15,6 +16,76 @@ make run
 # Navigate to http://localhost/, if you get a 502 error postgres likely has not been initialized yet,
 #   try again in a few seconds
 ```
+Use `docker-compose down` to stop hosting when you are done.
+
+## Local Development
+1) Create a python virtualenv, activate it, and install the package requirements:
+```bash
+# Clone the repo if you have not already.
+git clone https://github.com/chronoscio/backend
+
+# Navigate to the backend repo.
+cd backend
+
+# Begin a virtual environmnet.
+virtualenv venv
+
+# Activate the virtual environment.
+source venv/bin/activate
+
+# Install the project requirements manually.
+pip install -r config/requirements.txt
+
+# Create env files and note that you will need to modify them later.
+mv django.env.sample django.env
+mv postgres.env.sample postgres.env
+```
+2) Create an SPA (Single Page Application) [here](https://manage.auth0.com/#/applications).
+3) Go to the Settings of your SPA put your Client ID and Client Secret into `django.env` along with the website domain.
+```bash
+API_IDENTIFIER={IDENTIFIER}
+AUTH0_DOMAIN={PROJECT.auth0.com}
+AUTH0_CLIENT_ID={Client ID}
+AUTH0_CLIENT_SECRET={Client Secret}
+```
+4) Install and run [Postgresql](https://www.postgresql.org/docs/9.3/static/tutorial-install.html) (AKA postgres) locally. You should not need to worry about creating any accounts. You will also need to make sure you have set up [geodjango](https://docs.djangoproject.com/en/stable/contrib/gis/install/).  (TODO: Test this on wiped setups to find out specific steps to do this properly.)
```

Comment:
geodjango link broken (https://docs.djangoproject.com/en/stable/ref/contrib/gis/install/ is correct)